po/pt.po

```diff
@@ -1,21 +1,22 @@
 # GnuCash 2.6.15 pt_PT
 # Copyright (C) 2017
 # This file is distributed under the same license as the Gnucash package.
-# Pedro Albuquerque <palbuquerque73@gmail.com>, 2015, 2016, 2017.
+# Pedro Albuquerque <palbuquerque73@gmail.com>, 2015, 2016, 2017, 2018.
 #
 msgid ""
 msgstr ""
 "Project-Id-Version: GnuCash 2.6.15\n"
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2018-01-13 20:39+0100\n"
```

Comment:
You did not follow
https://wiki.gnucash.org/wiki/Translation#Get_a_fresh_template or
https://wiki.gnucash.org/wiki/Translation#Updating_an_existing_.po_file

So I have run make pot; msgmerge ... in the next commit resulting in
5082 translated messages, 114 fuzzy translations, 43 untranslated
messages.

Feel free to fix them in another PR.

Thank you very much!